bindings/java/mongocrypt/src/main/java/com/mongodb/crypt/capi/MongoCryptImpl.java

```diff
@@ -184,6 +188,11 @@ class MongoCryptImpl implements MongoCrypt {
             }
         }
 
+        options.getSearchPaths().forEach(p -> mongocrypt_setopt_append_csfle_search_path(wrapped, new cstring(p)));
+        if (options.getExtraOptions().containsKey("csflePath")) {
```

Comment:
Just a heads up in case you aren’t aware (this was only decided late last week), the `csfle` naming is going to be undone in the very near future ([WRITING-11210](https://jira.mongodb.org/browse/WRITING-11210))

Reply:
@kevinAlbs should we wait to merge this PR until the name change?

Reply:
Yes. The rename in libmongocrypt will be done shortly in MONGOCRYPT-434.

Reply:
Updated